### DISCUSSION TITLE
An account repeatedly stars the same repository

### DISCUSSION THREAD

**Discussion by Bachamht** [community/community#110583] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

After I star a repository using an account, I delete the account. However, the number of stars for that repository does not decrease accordingly, and when I recreate the account with the same email address, I can star again!

**Comment by Dell66665** [comment]
De

**Reply by Dell66665** [reply]
De

**Comment by Bobstar55** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> After I star a repository using an account, I delete the account. However, the number of stars for that repository does not decrease accordingly, and when I recreate the account with the same email address, I can star again!

**Comment by fmkamauer** [comment]
I'm

**Comment by fmkamauer** [comment]
1
